symengine/complex_double.h

```diff
@@ -67,50 +63,53 @@ class ComplexDouble : public Number {
     /*! Add ComplexDoubles
      * \param other of type Integer
      * */
-    RCP<const Number> addcomp(const Integer &other) const {
+    RCP<const Number> addcomp(const Integer &other) const
+    {
         return make_rcp<const ComplexDouble>(i + mp_get_d(other.i));
     }
 
```

Comment:
I think this is too much. For ex, I have to check that the column limit is 115 to figure out whether to break the line or not (Or run the script). 

Is it possible to make a setting in clang-format to only change single line functions into a separate line if the limit exceeds and not vice versa (change into a single line function if the limit doesn't exceed.)